packages/apps/hr/src/screens/ExpenseListScreen.js

```diff
@@ -159,7 +163,7 @@ const ExpenseListScreen = ({navigation}) => {
   );
 
   return (
-    <Screen removeSpaceOnTop={true}>
+    <FocusScreen removeSpaceOnTop={true} fetcher={refresh}>
```

Comment:
I think refresh should be on both types of expenses or at least on user's expenses as it's the most important on this screen.